tests/e2e-tests/specs/stock-panel.test.js

```diff
@@ -0,0 +1,153 @@
+
+		// Make sure we're on the add order page.
+		await expect( page.title() ).resolves.toMatch( 'Add new product' );
+
+		// Set product data.
+		await expect( page ).toFill( '#title', 'Low in stock product' );
+		await clickTab( 'General' );
+		await expect( page ).toFill( '#_regular_price', '9.99' );
+		await clickTab( 'Inventory' );
+		await expect( page ).toClick( '#_manage_stock' );
+		await page.waitForSelector( '#_stock' );
+		await expect( page ).toFill( '#_stock', '1' );
+
+		// Publish product.
+		await Promise.all( [
+			verifyAndPublish(),
+			page.waitForNavigation( { waitUntil: 'networkidle0' } ),
+		] );
+
+		// Verify that the stock panel indicates unread.
+		await page.waitForSelector( '#activity-panel-tab-stock.has-unread' );
+
+		// Open stock panel.
+		await expect( page ).toClick( '#activity-panel-tab-stock' );
+		await page.waitForSelector( '.woocommerce-stock-activity-card:not(.is-loading)' );
+
+		// Verify the low in stock product is displayed.
+		const stockCard = await page.$( '.woocommerce-stock-activity-card' );
+		await expect( stockCard ).toMatchElement( '.woocommerce-activity-card__title', { text: 'Low in stock product' } );
+	} );
+
+	it( 'can update inventory level', async () => {
+		// Open dashboard.
+		await page.goto( baseUrl + 'wp-admin/admin.php?page=wc-admin', {
+			waitUntil: 'networkidle0',
+		} );
+
+		// Open stock panel.
+		await expect( page ).toClick( '#activity-panel-tab-stock' );
+		await page.waitForSelector( '.woocommerce-stock-activity-card:not(.is-loading)' );
+
+		// Verify the low in stock product is displayed.
+		let stockCard = await page.$( '.woocommerce-stock-activity-card' );
+		await expect( stockCard ).toMatchElement( '.woocommerce-activity-card__title', { text: 'Low in stock product' } );
+
+		// Modify the inventory level.
+		await expect( stockCard ).toClick( 'button', { text: 'Update stock' } );
+
+		// We apparently need to reselect the card when the innerHTML changes.
+		stockCard = await page.$( '.woocommerce-stock-activity-card' );
+
+		await page.waitForSelector( '.woocommerce-stock-activity-card__edit-quantity input' );
+		await expect( stockCard ).toFill( 'input[type="number"]', '5' );
+		await expect( stockCard ).toClick( 'button', { text: 'Save' } );
+
+		// Refresh the page.
+		// NOTE: we should be able to just close and reopen the stock panel.
+		// There seems to be an issue where the panel doesn't recognize
+		// an empty array response as "no low stock items".
+		await page.reload( { waitUntil: 'networkidle0' } );
+
+		// Open stock panel.
+		await page.waitForSelector( '#activity-panel-tab-stock:not(.has-unread)' );
+		await expect( page ).toClick( '#activity-panel-tab-stock' );
+		await page.waitForSelector( '.woocommerce-empty-activity-card' );
+
+		// Verify there are no more low stock products.
+		await expect( page ).toMatchElement( '.woocommerce-layout__activity-panel-header-title', { text: 'No products with low stock' } );
+	} );
+} );
```

Comment:
With `tests/e2e-tests/specs/activate-wc-admin.test.js` should this file be removed to avoid duplicaton with Core?

Reply:
I'm not sure where the duplication with core is.. can you elaborate?